aip_site/support/assets/js/global.js

```diff
@@ -58,6 +58,11 @@ $.when($.ready).then(() => {
       .addClass(`spec-${directive.split(' ')[0]}`);
   }
 
+  // Make "reviewing sections" of approved AIPs show a badge.
+  let reviewing = $('p').filter((_, el) => $(el).text() === '[^reviewing]');
```

Comment:
Don't we want to find only the badges that are underneath a h3 ?

```suggestion
  let reviewing = $('h3 + p').filter((_, el) => $(el).text() === '[^reviewing]');
```